### ISSUE TITLE
Does STL Import work for anyone?

### ISSUE THREAD

**Issue by onsombal** [xibyte/jsketcher#79]
Has anyone gotten STL Import to work?

STL Export works fine for me.

**Comment by Evil-Spirit** [comment]
@mmiscool

**Comment by Evil-Spirit** [comment]
@mmiscool Let me introduce https://github.com/NoteCAD/NoteCAD

**Comment by zeser78** [comment]
No, Do you find a way to import?

**Comment by onsombal** [comment]
No, nothing new since @mmiscool replied that it's not working. I'm assuming it must be on the road map. It would be such a good feature!

**Comment by mmiscool** [comment]
Right now BREP and step file import/export is higher on the priority list. With much of the current work being done and the switch to using BREP rather than meshes for booleans. Using geometry from STL files will require some work where BREP is used for any new geometry and modifications and a secondary process to mesh the BREP solids and perform mesh booleans on the original STL. It is on the radar but not a priority at the moment.
